src/styles/watch.scss

```diff
@@ -0,0 +1,47 @@
+.watch {
+  width: 500px;
+  height: 500px;
+  border-radius: 50%;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: #3a2e2e;
+}
+
+.watch__center {
+  width: 20px;
+  height: 20px;
+  border-radius: 50%;
+
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  background: linear-gradient(229deg, #fbff00, #adaa16, #c7d413);
+  background-size: 240% 240%;
+  animation: gradient-animation 16s ease infinite;
+}
+
+.watch__arrows-minute {
+  height: 200px;
+  width: 18px;
+  background-color: #464861;
+  animation: hand-rotate 3600s infinite steps(60);
```

Comment:
It would be great to create variables for all main values (sizes, animation duration etc)